core/drivers/pm/sam/pm_suspend.S

```diff
@@ -327,12 +327,29 @@ SUSPEND_FUNC(at91_pm_suspend_in_sram)
 	mov	tmp1, #0
 	mcr	p15, 0, tmp1, c7, c10, 4
 
+	/* Flush tlb. */
+	mov	tmp1, #0
+	mcr	p15, 0, tmp1, c8, c7, 0
```

Comment:
Please use the `write_tlbiall` macro from the generated `out/core/include/generated/arm32_sysreg.S` instead.
Do you need an `isb` after this?

Reply:
Updated, thanks.